### ISSUE TITLE
roadmap

### ISSUE THREAD

**Issue by docgonzo2015** [CBot-official/CBot-Tcl#1]
Hi!
U have a roadmap for the bot?
Maybe i can help

**Comment by ComputerTechDev** [comment]
Yeah, well the current plan is to create a Basic IRC bot in each popular language, and then we can work on extra modules exactly the same per language for CBot, if you're interested in a language we've already been working on(Tcl, Python, Nim), feel free to create a pull request or if you'd like to become a member, let us know. We'd love more developers :)
